scss/utilities/_borders.scss

```diff
@@ -54,6 +54,10 @@
   border-radius: 50% !important;
 }
 
+.rounded-pill {
+  @include rounded-pill(1.5rem);
```

Comment:
The value is not enough for the documentation page:
![image](https://user-images.githubusercontent.com/11559216/46247950-2b9e0880-c413-11e8-8b33-8189d39e72d4.png)

Reply:
This is why it was originally `3.125rem` which is more reliable.